stl/debugger/STL.natvis

```diff
@@ -1680,6 +1679,52 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <Type Name="std::timed_mutex">
+    <AlternativeType Name="std::recursive_timed_mutex"/>
+    <DisplayString>{_My_mutex}</DisplayString>
+    <Expand>
+      <Item Name="[locking_thread_id]" Condition="sizeof(void *) == 4u &amp;&amp; *(int *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 44) != 0">*(long *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 40)</Item>
+      <Item Name="[locking_thread_id]" Condition="sizeof(void *) == 8u &amp;&amp; *(int *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 76) != 0">*(long *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 72)</Item>
+      <Item Name="[ownership_levels]" Condition="sizeof(void *) == 4u &amp;&amp; *(int *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 44) != 0">*(int *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 44)</Item>
+      <Item Name="[ownership_levels]" Condition="sizeof(void *) == 8u &amp;&amp; *(int *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 76) != 0">*(int *)((char *)(&amp;(_My_mutex._Mtx_storage)) + 76)</Item>
+      <Item Name="[My owner]" Optional="true">_My_owner</Item>
+    </Expand>
+  </Type>
+
+  <Type Name="std::lock_guard&lt;*&gt;">
+    <DisplayString>{_My_mutex}</DisplayString>
+  </Type>
+
+  <Type Name="std::unique_lock&lt;*&gt;">
+    <DisplayString>{*_Pmtx}</DisplayString>
+    <Expand>
+      <Item Name="owns">_Owns</Item>
+    </Expand>
+  </Type>
+
+  <Type Name="std::scoped_lock&lt;*&gt;">
+    <DisplayString>{_MyMutex}</DisplayString>
+  </Type>
+
+  <Type Name="std::scoped_lock&lt;*,*&gt;">
+    <DisplayString>{_MyMutexes}</DisplayString>
+  </Type>
+
+  <Type Name="std::defer_lock_t">
+    <DisplayString>defer lock</DisplayString>
+  </Type>
+
+  <Type Name="std::try_to_lock_t">
+    <DisplayString>try to lock</DisplayString>
+  </Type>
+
+  <Type Name="std::adopt_lock_t">
+    <DisplayString>adopt lock</DisplayString>
+  </Type>
+
+  <Type Name="std::once_flag">
+    <DisplayString>{(int)_Opaque}</DisplayString>

```

Comment:
Can we decipher this (using some undocumented knowledge)?

Reply:
I don't have knowledge Im going to try to investigate more about this

Reply:
I didn't found anything useful. I'm thinking on changing the display string to just `Once flag`. Any thoughts on that?

Reply:
As per discussion in Discord channel, this should be left as it is currently in this PR, exposing `_Opaque`, but not its details.